components/Input/index.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+
+const Input = ({ id, type, dataTestId, required, className }) => (
+  <input
+    className={`w-full h-full bg-dark text-white text-base-large rounded-xl px-3 pt-[19px] pb-[15px] focus:outline-none focus:shadow-none ${className}`}
+    id={id}
+    data-test-id={dataTestId}
+    type={type}
+    required={required}
+  />
+);
```

Comment:
```suggestion
const Input = ({ id, className, attributes }) => (
  <input
    className={`w-full h-full bg-dark text-white text-base-large rounded-xl px-3 pt-[19px] pb-[15px] focus:outline-none focus:shadow-none ${className}`}
    id={id}
    {...attributes}
  />
);
```

Using the spread operator in this case could be interesting so that later on, you could use the `disabled`, or `name` attributes without changing this component.
If you apply this suggestion, there is 2 points of attention:
- Need to rename the `dataTestId` prop to `data-test-id` when using this component in other files
- Need to research what is the appropriate `propTypes` for that ([from a quick search](https://stackoverflow.com/a/44326022/1914241), it seems we do not need to include the `attributes` prop in the prop types...)

Reply:
loved the idea of spread operators here, used it in both `Input` and `Button` components in ec0d501e73cda2a9b2acc0fbacb4ce100eaeba50